coremltools/converters/mil/frontend/torch/test/test_executorch_e2e.py

```diff
@@ -92,6 +94,139 @@ def forward(self, a, x, b):
             backend=("mlprogram", "fp16"),
         )
 
+    def test_softmax(self):
+        class LinearModule(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.softmax = torch.nn.Softmax()
+
+            def forward(self, x):
+                z = self.softmax(x)
+                return z
+
+        model = LinearModule()
+        model.eval()
+
+        self.run_compare_torch(
+            [(2, 2)], model, frontend=TorchFrontend.EDGEIR, backend=("mlprogram", "fp16")
+        )
+
+    @pytest.mark.xfail(reason="numerical error")
+    def test_deeplab_v3(self):
+        model = torchvision.models.segmentation.deeplabv3_resnet50(
+            weights=torchvision.models.segmentation.deeplabv3.DeepLabV3_ResNet50_Weights.DEFAULT
+        )
+        model.eval()
+
+        self.run_compare_torch(
+            [(1, 3, 224, 224)], model, frontend=TorchFrontend.EDGEIR, backend=("mlprogram", "fp16")
+        )
+
+    def test_edsr(self):
+        model = torchsr.models.edsr_r16f64(2, True)
+        model.eval()
+
+        self.run_compare_torch(
+            [(1, 3, 224, 224)], model, frontend=TorchFrontend.EDGEIR, backend=("mlprogram", "fp16")
+        )
+
+    def test_enformer_transcribe(self):
+        class EmformerRnntTranscriberExample(torch.nn.Module):
```

Comment:
It looks like there is a lot of duplicated code between this method and the next two methods. Are the constructors for these classes identical? If so, you probably should have them repeated.

Reply:
This is to be consistent with [ExecuTorch](https://github.com/pytorch/executorch/blob/main/examples/models/emformer_rnnt/model.py#L27). I guess they would like to keep each model independent, so more flexible to revise in the (probably near) future